src/sentry/services/hybrid_cloud/app/impl.py

```diff
@@ -157,13 +164,16 @@ def serialize_api(self, serializer: Optional[None]) -> Serializer:
         def apply_filters(
             self, query: QuerySet[SentryAppInstallation], filters: SentryAppInstallationFilterArgs
         ) -> QuerySet[SentryAppInstallation]:
-            # filters["status"] = SentryAppInstallationStatus.INSTALLED
             if "installation_ids" in filters:
                 query = query.filter(id__in=filters["installation_ids"])
+            if "app_ids" in filters:
+                query = query.filter(sentry_app_id__in=filters["app_ids"])
```

Comment:
Am I missing something, or was this always just broken?

Reply:
It certainly looks like it wasn't implemented before, but also wasn't used either 🙃